cmd/nodecmd/join.go

```diff
@@ -61,7 +62,16 @@ avalanche node status`,
 	return cmd
 }
 
-func parseBootstrappedOutput(filePath string) (bool, error) {
+func printJSONOutput(byteValue []byte) error {
+	var prettyJSON bytes.Buffer
+	if err := json.Indent(&prettyJSON, byteValue, "", "   "); err != nil {
+		return err
+	}
+	ux.Logger.PrintToUser(prettyJSON.String())
+	return nil
+}
+
+func parseBootstrappedOutput(filePath string, printOutput bool) (bool, error) {
```

Comment:
is this printOutput for debugging no? maybe we can use log level? cc @dasconnor

Reply:
instead of a boolean

Reply:
Yeah, I think this pattern is good